wordfreq_builder/wordfreq_builder/ninja.py

```diff
@@ -160,11 +156,14 @@ def twitter_deps(input_filename, slice_prefix,
         count_file = wordlist_filename('twitter', language, 'counts.txt')
 
         if language == 'ja':
-            mecab_token_file = wordlist_filename('twitter', language, 'mecab-tokens.txt')
-            add_dep(lines, 'tokenize_japanese', combined_output, mecab_token_file)
-            add_dep(lines, 'count', mecab_token_file, count_file, extra='wordfreq_builder/tokenizers.py')
-        else:
-            add_dep(lines, 'count', combined_output, count_file, extra='wordfreq_builder/tokenizers.py')
+            mecab_token_file = wordlist_filename(
```

Comment:
Style thing: if the opening parenthesis is at the end of a line, the closing parenthesis should be at the beginning of one.